integration/spark/build.gradle

```diff
@@ -76,14 +79,18 @@ dependencies {
     implementation 'org.javassist:javassist:3.27.0-GA'
     implementation 'com.github.ok2c.hc5:hc5-async-json:0.2.1'
     implementation 'org.apache.httpcomponents.client5:httpclient5:5.0.3'
-    implementation "com.fasterxml.jackson.core:jackson-databind:${jacksonVersion}"
-    implementation "com.fasterxml.jackson.core:jackson-core:${jacksonVersion}"
-    implementation "com.fasterxml.jackson.datatype:jackson-datatype-jsr310:${jacksonVersion}"
-    implementation "com.fasterxml.jackson.module:jackson-module-scala_2.11:${jacksonVersion}"
+    implementation "com.fasterxml.jackson.datatype:jackson-datatype-jsr310:${jacksonDatatypeVersion}"
```

Comment:
Not sure that we need it, I guess we can just create our own ZoneDateTime converter